BuildAssets/test-linux-packages.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script requires environment variables to run:
+#   OCTOPUS_CLI_SERVER, OCTOPUS_CLI_API_KEY, OCTOPUS_SPACE, OCTOPUS_EXPECT_ENV
+# You must also supply either:
+#   TEST_QUICK (to skip some distributions)
+# or
+#   REDHAT_SUBSCRIPTION_USERNAME, REDHAT_SUBSCRIPTION_PASSWORD (to support RHEL tests)
+
+TEST_DEB_SH='
```

Comment:
Any reason you didn't do these as separate files?

Reply:
No strong reason. I think the self-containedness makes it robust and easier to understand but those blocks are a little harder to read, I admit.